cmd/cluster-agent/subcommands/start/command.go

```diff
@@ -345,6 +346,12 @@ func start(log log.Component, config config.Component, taggerComp tagger.Compone
 		}()
 	}
 
+	if pkgconfig.Datadog.GetBool("language_detection.enabled") {
```

Comment:
I think we should not rely here on admission controller, because in that case we will not be able to evaluate/test language detection while also disabling the admission controller.

I think this capability is important, at least internally, to be able to see what languages are being detected, how annotations are being patched and also evaluate the impact on load on k8s api server. All this should be done while admission controller is not injecting libraries of detected languages.

WDYT?